liquidsoap/radio.liq

```diff
@@ -237,13 +240,24 @@ end
 
 live_dj := map_metadata(new_meta, !live_dj,update=false)
 
+live_video := input.ffmpeg(id="main_video", max_buffer=60.0, "https://viz.streampusher.com:16666/hls/datafruits.m3u8")
+vj_video := input.ffmpeg(id="vj_video", max_buffer=60.0, "https://viz.streampusher.com:16666/vj/datafruits.m3u8")
+
+live_video_audio := audio_to_stereo(drop_video(!live_video))
 # live_dj := pipe(process="./stereo_tool_cmd_64 - - -s ./stereotool.sts -k '#{stereo_tool_key}'", buffer=10.0, replay_delay=10.0, !live_dj)
 
 # output.dummy(fallible=true, on_start=dj_dummy_started, on_stop=dj_dummy_stopped, !live_dj)
 output.file(%mp3, "/home/liquidsoap/recordings/#{radio_name}-#{current_dump_filename}.mp3", !live_dj, fallible=true, append=true)
 
 source = fallback(id="fallback",track_sensitive=false,
-                  [!live_dj,scheduled_shows,mksafe(backup_playlist)])
+                  [!live_video_audio, !live_dj,scheduled_shows,mksafe(backup_playlist)])
+
+single_video = noise()
+
+video_source = fallback(id="video_fallback",track_sensitive=false,
+                        [!live_video,
+                         mux_audio(drop_audio(!vj_video), audio=source),
+                         mksafe(mux_audio(single_video, audio=source))])
```

Comment:
Error 10: A source cannot belong to two clocks (mux_audio_1.child[],
mux_audio_0.child[]).
The command '/bin/sh -c eval $(opam env) && liquidsoap --check radio.liq' returned a non-zero code: 1

Reply:
The problem here is that the muxing operator wants control over the sources being muxed so make sure that it can pull data from both sources at the rate that's needed to synchronize audio and video content.

The `buffer` operator should be available to decorelate this so you can pass the same source into two different muxers. Something like this:
```
video_source = fallback(id="video_fallback",track_sensitive=false,
                        [!live_video,
                         mux_audio(drop_audio(!vj_video), audio=buffer(source)),
                         mksafe(mux_audio(single_video, audio=buffer(source)))])
```
This operator is in need of some love so let us know if you have any issue with it.

Reply:
Yeah